msal/src/main/java/com/microsoft/identity/client/internal/controllers/OperationParametersAdapter.java

```diff
@@ -51,6 +51,20 @@
 
     private static final String TAG = OperationParameters.class.getName();
 
+    public static OperationParameters createOperationParameters (
```

Comment:
Who is calling this?  OperationParameters is the base class for AcquireToken and AcquireTokenSilent operation parameters.

Reply:
This is used to create the OperationParameters for account commands (LoadAccountsCommand and RemoveAccountCommand). I can create a subclass from OperationParameters but it does not have any additional properties to add though.